lib/rdoc/markup/to_html_crossref.rb

```diff
@@ -151,6 +151,8 @@ def link name, text, code = true, rdoc_ref: false
       label = $'
     end
 
+    return name if name == text && @options.autolink_excluded_words&.include?(name)
```

Comment:
Does this work with `C1[rdoc-ref:C1]`?
(I think that this should be linked.)

Reply:
Isn't the last assertion in `test_convert_CROSSREF_ignored_excluded_words` that case?

Reply:
> Does this work with C1[rdoc-ref:C1]?

Yes in that case RDoc would still link it. And it's covered by a test as well.

Reply:
(`name == text` was needless. If `name == text` exists, `Constant[rdoc-ref:C1]` works but `C1[rdoc-ref:C1]` doesn't work. Because both of `name` and `text` are `C1` with `C1[rdoc-ref:C1]`.)